packages/react-components/react-portal/library/src/components/Portal/Portal.test.tsx

```diff
@@ -6,14 +6,11 @@ import * as React from 'react';
 import { Portal } from './Portal';
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33174&commitId=5839350e35fe226c066386cd908a36d8d2cd66d8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|8| Changed|

</details>


  <details><summary><b>Dialog</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Nested RTL.chromium.png|12727| Changed|
|Dialog.nested.chromium.png|12768| Changed|
|Dialog.Nested High Contrast.chromium.png|9003| Changed|
|Dialog.Nested Dark Mode.chromium.png|4761| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>